lm_eval/tasks/redlite/rt-cogensumm.yaml

```diff
@@ -0,0 +1,12 @@
+task: rt-cogensumm

```

Comment:
Regarding task naming: could we prefix `redlite_` to all these tasks' names for clarity?

(most tasks in the repo also tend to favor `_` instead of `-`, but that's less important)

Reply:
done